### PR TITLE
Add chromatic

### PR DESCRIPTION
**Pull request by torztomasz**
Resolves L2B-2887

Right now after every push the Chromatic build will be deployed and the changes will be compared with the baselines. It is required to accept the changes to merge PR. Turbosnap feature detects what files have changed which can save us some money on the pricing plan.

### CODE REVIEW

**Comment by antooni** [comment]
Please in the description provide the proposed workflow of the new viz-reg, when it will run? should it be required? why merge queues were needed? what are the problems with pricing and how the turbo snapshot can help fix that

File: .github/workflows/chromatic.yml
Comment (antooni) [inline .github/workflows/chromatic.yml:1]: Is it copied from some template or was it written by you?

File: .github/workflows/chromatic.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: "Chromatic"
+
+# It would be nice to use the @chromeui/action to build storybook and deploy to chromatic
```
Comment (antooni) [inline .github/workflows/chromatic.yml:3]: Is this comment still valid?

Comment (torztomasz) [reply .github/workflows/chromatic.yml:3]: yup

File: .github/workflows/chromatic.yml
Comment (torztomasz) [reply .github/workflows/chromatic.yml:1]: written by me, template was not working

File: packages/frontend/package.json [hunk 2]
```diff
@@ -50,16 +51,20 @@
   },
   "devDependencies": {
     "@babel/core": "^7.18.10",
-    "@storybook/addon-actions": "^7.0.9",
-    "@storybook/addon-essentials": "^7.0.9",
+    "@storybook/addon-actions": "7.0.9",
```
Comment (antooni) [inline packages/frontend/package.json:54]: is this a desired change? if so, why?

File: packages/frontend/package.json [hunk 3]
```diff
@@ -69,10 +74,13 @@
...
+    "storybook": "7.0.9",
+    "storybook-addon-pseudo-states": "^2.1.2",
     "tailwindcss": "^3.1.0",
     "ts-node": "^10.9.1",
-    "vite": "^3.0.9"
-  }
+    "vite": "^3.0.9",
+    "vite-plugin-turbosnap": "^1.0.3"
+  },
+  "readme": "ERROR: No README data found!",
```
Comment (antooni) [inline packages/frontend/package.json:84]: ?

File: packages/frontend/package.json [hunk 4]
```diff
@@ -69,10 +74,13 @@
...
+  "_id": "@l2beat/frontend@0.1.0"
```
Comment (antooni) [inline packages/frontend/package.json:85]: ?

File: packages/frontend/.eslintrc.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  extends: '../../.eslintrc.json',
+  extends: ['../../.eslintrc.json', 'plugin:storybook/recommended'],
```
Comment (antooni) [inline packages/frontend/.eslintrc.js:2]: why is this needed?

**Review by antooni — CHANGES_REQUESTED** [review]
GJ, left small comments, please resolve them

File: packages/frontend/package.json (hunk 2 above)
Comment (torztomasz) [reply packages/frontend/package.json:54]: yup, we want to keep the 7.0.9 version as the newer ones have some conflicts with our codebase

File: packages/frontend/package.json (hunk 3 above)
Comment (torztomasz) [reply packages/frontend/package.json:84]: actually, no clue - will delete that

File: packages/frontend/package.json (hunk 4 above)
Comment (torztomasz) [reply packages/frontend/package.json:85]: actually, no clue - will delete that

File: packages/frontend/.eslintrc.js (hunk 5 above)
Comment (torztomasz) [reply packages/frontend/.eslintrc.js:2]: to make use of the storybook eslint plugin

**Review by antooni — APPROVED** [review]
